prioritization.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+---
+
+# Prioritization guidelines
+
+This document describes the principles to be considered when introducing major functionality and sponsorship for [The Good Docs Project](https://thegooddocsproject.dev/). It should be read in conjunction with our [budget approval process](budget.html).
+
+**Version:** 1.0
+
+**Last updated:** August 2020
+
+## Prioritization guidelines
+
+We prioritize the following principles when considering budget allocation:
+
+*   We consider community engagement to be very important. It is an indicator of long term sustainability.
+*   We like to help the enablers within communities. These people often take on the tedious but important jobs. Or they help tackle and resolve difficult problems. They may be both old timers or fresh and enthusiastic new contributors.
+*   We prioritize ideas which have wide support from within our community forums. Ideas need to be debated within our forums and have gathered support before they will be considered for sponsorship.
+*   We prioritize high value ideas, ones which lift the collective effectiveness of our community.
+*   We like unblocking obstacles preventing the community from moving forward. Maybe a technical challenge, maybe a bunch of boring grunt work or it could be business-as-usual overheads.
+*   We prioritize ideas which are not already being solved elsewhere.
+*   We are wary of introducing a maintenance liability. We want material which is going to be adopted and maintained by the greater community.
+*   We look favourably on matching contributions. If someone else is prepared to sponsor your idea, it is a good indicator of a worthiness.
+*   We like to help owners of useful proprietary material who wish to share it under an open license. This could be by covering the business costs of sharing, or by counting the material as an in-kind contribution.
```

Comment:
What does "in-kind" mean in this context?

Reply:
it means payment in goods/materials rather than money

Reply:
Well, yes, I know that. What does it mean in context?